app/subscriber/src/components/sidebar/CustomSidebar.tsx

```diff
@@ -7,13 +7,30 @@ import { AdvancedSearch } from './advanced-search/AdvancedSearch';
 import { SelectableMenuItems } from './SelectableMenuItems';
 import * as styled from './styled';
 
+export interface ICustomSidebarProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
No need to change, but when I see this I wonder if a useContext makes sense, or redux.